src/PageTypes/BaseHomePage.php

```diff
@@ -62,14 +95,14 @@ public function getCMSFields() {
 				GridFieldConfig_RelationEditor::create()
 			);
 			$gridConfig = $gridField->getConfig();
-			$gridConfig->getComponentByType('GridFieldAddNewButton')->setButtonName(
+			$gridConfig->getComponentByType(GridFieldAddNewButton::class)->setButtonName(
 				_t('BaseHomePage.AddNewButton','Add new')
 			);
-			$gridConfig->removeComponentsByType('GridFieldAddExistingAutocompleter');
-			$gridConfig->removeComponentsByType('GridFieldDeleteAction');
+			$gridConfig->removeComponentsByType(GridFieldAddExistingAutocompleter::class);
+			$gridConfig->removeComponentsByType(GridFieldDeleteAction::class);
 			$gridConfig->addComponent(new GridFieldDeleteAction());
```

Comment:
I believe they `use Injectable` right, so should we change this from `new ` to `::create()`?

Reply:
They aren't in SS4. I suppose we could future proof if for SS5 by loading with injector now anyway though